samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -20,75 +25,156 @@ public static MauiApp CreateMauiApp()
+		services.AddTransient<SnackbarPage>();
+		services.AddTransient<ToastPage>();
+
+		// Add Behaviors Pages
+		services.AddTransient<CharactersValidationBehaviorPage>();
+		services.AddTransient<EmailValidationBehaviorPage>();
+		services.AddTransient<EventToCommandBehaviorPage>();
+		services.AddTransient<MaskedBehaviorPage>();
+		services.AddTransient<MaxLengthReachedBehaviorPage>();
+		services.AddTransient<MultiValidationBehaviorPage>();
+		services.AddTransient<NumericValidationBehaviorPage>();
+		services.AddTransient<ProgressBarAnimationBehaviorPage>();
+		services.AddTransient<RequiredStringValidationBehaviorPage>();
+		services.AddTransient<SetFocusOnEntryCompletedBehaviorPage>();
+		services.AddTransient<TextValidationBehaviorPage>();
+		services.AddTransient<UriValidationBehaviorPage>();
+		services.AddTransient<UserStoppedTypingBehaviorPage>();
+
+		// Add Converters Pages
+		services.AddTransient<BoolToObjectConverterPage>();
+		services.AddTransient<ByteArrayToImageSourceConverterPage>();
+		services.AddTransient<ColorsConverterPage>();
+		services.AddTransient<CompareConverterPage>();
+		services.AddTransient<DateTimeOffsetConverterPage>();
+		services.AddTransient<DoubleToIntConverterPage>();
+		services.AddTransient<EnumToBoolConverterPage>();
+		services.AddTransient<EnumToIntConverterPage>();
+		services.AddTransient<EqualConverterPage>();
+		services.AddTransient<ImageResourceConverterPage>();
+		services.AddTransient<IndexToArrayItemConverterPage>();
+		services.AddTransient<IntToBoolConverterPage>();
+		services.AddTransient<InvertedBoolConverterPage>();
+		services.AddTransient<IsListNotNullOrEmptyConverterPage>();
+		services.AddTransient<IsListNullOrEmptyConverterPage>();
+		services.AddTransient<IsStringNotNullOrEmptyConverterPage>();
+		services.AddTransient<IsStringNotNullOrWhiteSpaceConverterPage>();
+		services.AddTransient<IsStringNullOrEmptyConverterPage>();
+		services.AddTransient<IsStringNullOrWhiteSpaceConverterPage>();
+		services.AddTransient<ItemSelectedEventArgsConverterPage>();
+		services.AddTransient<ItemTappedEventArgsConverterPage>();
+		services.AddTransient<ListToStringConverterPage>();
+		services.AddTransient<MathExpressionConverterPage>();
+		services.AddTransient<MultiConverterPage>();
+		services.AddTransient<NotEqualConverterPage>();
+		services.AddTransient<StringToListConverterPage>();
+		services.AddTransient<TextCaseConverterPage>();
+		services.AddTransient<VariableMultiValueConverterPage>();
+
+		// Add Extensions Pages
+		services.AddTransient<ColorAnimationExtensionsPage>();
+
+		// Add Layouts Pages
+		services.AddTransient<UniformItemsLayoutPage>();
+
+		// Add Views Pages
+		services.AddTransient<CsharpBindingPopup>();
+		services.AddTransient<MultiplePopupPage>();
+		services.AddTransient<PopupAnchorPage>();
+		services.AddTransient<PopupPositionPage>();
+		services.AddTransient<XamlBindingPopup>();
+	}
+
+	static void RegisterViewModels(IServiceCollection services)
```

Comment:
Let's add `in` to ensure a future PR doesn't change `IServiceCollection`

```suggestion
	static void RegisterViewModels(in IServiceCollection services)
```